File: qlty-cli/tests/lang/java/basic.in/FunctionComplexity.java
class FunctionComplexity {
    void simple() {
    }

    void complex() {
        int bar = 42;
        if (bar > 0) {
            if (bar > 10) {
                if (bar < 20) {
                    if (bar % 2 == 0) {
                        if (bar % 3 == 0) {
                            System.out.println("Nested!");
                        }
                    }
                }
            }
        }
    }
}

File: qlty-cli/tests/lang/rust/basic.in/function_complexity.rs
fn simple() {}

fn complex() {
    if let Some(bar) = Some(42) {
        if bar > 10 {
            if bar < 20 {
                if bar % 2 == 0 {
                    if bar % 3 == 0 {
                        println!("Nested!");
                    }
                }
            }
        }
    }
}
